src/ashley/templates/forum_search/search.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'board_base.html' %}
```

Comment:
note that we could propose django-machina to add {% block %} sections in their templates so we can override them with more granularity.